conda_build_config.yaml

```diff
@@ -1,2 +1,2 @@
-MACOSX_DEPLOYMENT_TARGET:  # [osx]
-  - 11.0                   # [osx]
\ No newline at end of file
+c_stdlib_version:  # [osx]
+  - 11.0           # [osx]
```

Comment:
Are you sure you want to be changing things outside of `recipes` here?  Wouldn't it be better to do this as its own PR?

Reply:
@ocefpaf?

Reply:
Yep. Let's just revert this and ignore the linter for now. Whoever made this change in conda-smithy probably just forgot to update it here. I'm out of the loop in this change so I'd rather not touch it for now.

Reply:
@xhochy, it looks like you were involved in the PR that added this file:
https://github.com/conda-forge/staged-recipes/pull/24549
Any insight?

In any case, nothing that needs to be discussed here...